content/applications/hr/payroll/contracts.rst

```diff
@@ -0,0 +1,249 @@
+A contract form will appear and the information can be entered. Required fields are underlined in
+bold.
+
+Required fields
+---------------
+
+.. image:: contracts/required-fields.png
+   :align: center
+   :alt: New contract form to be filled in when creating a new contract.
+
+- :guilabel:`Contact Reference`: Type in the name or title for the contract, such as `John Smith
+  Contract`.
+- :guilabel:`Company`: Select the company the contract applies to by clicking on the drop-down
+  menu. A new company can be created by typing the name in the box, then clicking either
+  :guilabel:`Create` to create the new company, or :guilabel:`Create and Edit` to create the new
+  company and edit the company details.
+- :guilabel:`Salary Structure Type`: Select one of the salary structure types from the drop-down
+  menu. The default salary structure types are :guilabel:`Employee` or :guilabel:`Worker`. A new
+  salary structure type can be created by typing the name in the field.
+- :guilabel:`Start Date`: The date the contract will start. Choose a date by clicking on the drop
+  down menu, navigating to the correct month and year by using the arrow icons, then clicking on
+  the date.
+- :guilabel:`Working Schedule`: Select one of the working schedules from the drop-down menu.
+
+.. tip::
+   The :guilabel:`Working Schedule` drop-down menu displays all the working times for the selected
+   :guilabel:`Company`.
+   To modify or add to this list, go to :menuselection:`Payroll --> Configuration --> Working Times`
+   and either :guilabel:`Create` a new working time or click on an existing working time and
+   edit it by clicking :guilabel:`Edit`.
+
+Contract Details
+----------------
+
+.. image:: contracts/optional-fields.png
+   :align: center
+   :alt: Optional fields for a new contract.
+
+- :guilabel:`Employee`: Name of the employee that the contract applies to.
+- :guilabel:`Department`: The department the contract applies to.
+- :guilabel:`Job Position`: The specific job position the contract applies to.
+- :guilabel:`Contract Type`: Chose from :guilabel:`CDI`, :guilabel:`CDD`, or :guilabel:`PFI` from
+  the drop-down menu.
+
+   - :guilabel:`CDI` is an open-ended contract with just a start date but no end date.
+   - :guilabel:`CDD` is a contract with both a start date and an end date.
+   - :guilabel:`PFI` is a Belgian-specific contract used when hiring employees that need training,
+     and covers the training period specifically.
+
+- :guilabel:`End Date`: If the contract has a specific end date, enter it here. Chose a date by
+  clicking on the drop-down menu, navigating to the correct month and year by using the arrow
+  icons, then clicking on the date.
+- :guilabel:`HR Responsible`: If there is a specific person in HR that is responsible for the
+  contract, select the person from the drop-down menu.
+- :guilabel:`Analytic Account`: This field allows a link between the contract and a specific
+  analytic account for accounting purposes.
+
```

Comment:
Good idea- I was torn, since if the words were capitalized on the UI I was doing it to match, but since "contract details" is also a generic term, I think it works to keep it sentence case.